php/sqlcommenter-php/packages/sqlcommenter-laravel/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing to php-laravel
+
+The contributions for Sqlcommenter laravel should be made to this [repo](https://github.com/google/sqlcommenter)
+
+## Download Source
+
+```shell
+git clone https://github.com/google/sqlcommenter
+```
+
+## Install from source
+
+Add this to your `composer.json`
+```shell
+"repositories": [
+    {
+        "type": "path",
+        "url": "/full/or/relative/path/to/sqlcommenter-laravel/package"
```

Comment:
the suffix can imitate the laravel package. Maybe something like: `/<full or relative path to local repository>/php/sqlcommenter-php/packages/sqlcommenter-laravel/package` ?